.gitignore

```diff
@@ -1,72 +1,10 @@
 node_modules
 dist
+coverage
 sandbox/public/alloy.js
-sandbox/public/alloy.min.js
 sandbox/build
 .sonarlint
 .env
 allure
 logs
-*.log
-npm-debug.log*
-
-.DS_Store
-
-sauceconnect
-
-# Runtime data
-pids
-*.pid
-*.seed
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-
-# nyc test coverage
-.nyc_output
-
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-# node-waf configuration
-.lock-wscript
-
-# Compiled binary addons (http://nodejs.org/api/addons.html)
-build/Release
-
-# Dependency directories
-node_modules
-jspm_packages
-bower_components
-
-# Optional npm cache directory
-.npm
-
-# Optional REPL history
-.node_repl_history
-
-# Build result
-dist
-
-# Documentation
-.sass-cache
-_site
-
-# TypeScript definitions
-typings
-
-# IDEA
-.idea/
-
-# Rollup plugin script
-.rts2_cache/
-.rpt2_cache/
-
-TMP_DIR/
-
-bin/
-
-deploy_key
```

Comment:
Anything not project-specific but is instead system-specific should be added to the global gitignore on your machine. Some of these things look system-specific while other look unrelated to our project entirely (e.g., bower, sass, typescript). https://gist.github.com/subfuzion/db7f57fff2fb6998a16c